handlers/main.yml

```diff
@@ -17,4 +17,12 @@
   ansible.builtin.service:
     name: apache2
     state: restarted
-  when: roundcube_apache | bool
+  when: ansible_facts.services is search("apache2.service")
+  notify:
+    - Restart php-fpm
```

Comment:
Technically we don't have to restart apache if `php-fpm` is used. Also, you might run `php-fpm` with another reverse proxy. So I'd suggest to just always notify `Restart apache`  and `Restart php-fpm` when necessary.

Reply:
Thank you for your feedback @doobry-systemli . What do you think of this more generic solution? Apache2 might be restarted without a reason, but I don't see this as a big problem?